.github/workflows/test-build.yml

```diff
@@ -98,32 +101,85 @@ jobs:
       - name: Create version file
         run: git rev-parse --short HEAD > .version
 
-      - name: Build circ-scripts image
+      - name: Build circ-scripts image (linux/amd64)
```

Comment:
Instead of copy pasting these steps you should be able to use a build matrix here.

Reply:
```yaml
    strategy:
      fail-fast: false
      matrix:
        platform: ["linux/amd64", "linux/arm64"]
```